apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/java/gov/cms/model/dsl/codegen/plugin/model/validation/EnumExistsInSameMappingValidatorTest.java

```diff
@@ -0,0 +1,70 @@
+package gov.cms.model.dsl.codegen.plugin.model.validation;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import gov.cms.model.dsl.codegen.plugin.model.EnumTypeBean;
+import gov.cms.model.dsl.codegen.plugin.model.MappingBean;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.Optional;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+/** Unit tests for EnumExistsInSameMappingValidator. */
+public class EnumExistsInSameMappingValidatorTest {
+  /** Used to mack static method called by the validator. */
```

Comment:
everything is so good, I feel like I need to find something just to show that I've gone through everything....of course if I had a nickel for every spelling error I made, I probably wouldn't be reviewing the code right now!

'mack static'

Reply:
lol, great catch!  The font in my IDE made that really hard to spot.